test/unit/testApplicantAliasReason.js

```diff
@@ -14,4 +14,33 @@ describe('ApplicantAliasReason', () => {
             done();
         });
     });
+
+    describe('handlePost()', () => {
+        let ctx;
+        let errors;
+        it('should delete otherReason when the aliasReason is not set to other', (done) => {
+            ctx = {
+                aliasReason: 'Divorce',
+                otherReason: 'because I wanted to'
+            };
+            errors = {};
+            [ctx, errors] = ApplicantAliasReason.handlePost(ctx, errors);
+            expect(ctx).to.deep.equal({aliasReason: 'Divorce'});
+            done();
+        });
+
+        it('should not remove otherReason from the ctx when the aliasReason is not set to other', (done) => {
```

Comment:
This test is checking when `otherReason` is set to `other` so I think this test description should be

 `should not remove otherReason from the ctx when the aliasReason is set to other`